label_sync/cluster/label_sync_cron_job.yaml

```diff
@@ -15,15 +15,36 @@
 apiVersion: batch/v2alpha1
 kind: CronJob
 metadata:
-  name: label_sync_cron_job
+  name: label-sync-cron-job
 spec:
   schedule: "17 23 * * 0"    # Every Sunday 23:17 / 11:17PM
   jobTemplate:
     spec:
       template:
         spec:
           containers:
-            - name: label_sync
-              image: gcr.io/k8s-prow/label_sync:0.100
-          restartPolicy: OnFailure
+            - name: label-sync
+              image: gcr.io/k8s-testimages/label_sync:v20171121-90506fba
+              args:
+              - --config=/etc/config/labels.yaml
+              - --dry-run=false
+              - --orgs=kubernetes
+              - --token=/etc/github/oauth
+              imagePullPolicy: Always
```

Comment:
This is unnecessary because the image is tagged with the git sha. (This also applies to the job config).

Reply:
Done